src/components/shared/C3Chart/C3Chart.js

```diff
@@ -0,0 +1,21 @@
+import c3 from 'c3';
+import React, { useState, useEffect } from 'react';
+
+const C3Chart = ({ config, element }) => {
+  const [chart, setChart] = useState();
+
+  const generateChart = (config, element) => {
+    const build = Object.assign({ bindto: '#' + element }, config);
+    setChart(c3.generate(build));
+  };
+
+  useEffect(() => {
+    if (!chart) {
```

Comment:
If data changes, does this graph get updated? Just to know.

Reply:
now changed